sdk/src/Core/Amazon.Util/AWSSDKUtils.cs

```diff
@@ -1643,6 +1643,22 @@ public static string CompressSpaces(string data)
             return stringBuilder.ToString();
         }
 
+        /// <summary>
+        /// Extracts the operation name from a given request name.
+        /// </summary>
+        /// <param name="requestName">The name of the request from which the operation name is to be extracted.</param>
+        /// <returns>
+        /// The operation name if the request name ends with "Request"; otherwise, returns the original request name.
+        /// </returns>
+        public static string ExtractOperationName(string requestName)
```

Comment:
Can this be `internal static string`  so we don't have to expose it to customers?

Reply:
Switched it to internal in place, since this utilities class contain other internal methods.